flexmeasures/data/scripts/cli_tasks/data_add.py

```diff
@@ -280,11 +289,20 @@ def add_beliefs(
             bdf, expunge_session=True, allow_overwrite=False, commit_transaction=True
         )
     except IntegrityError as e:
```

Comment:
I would simply pass `allow_overwrite` on to timely_beliefs. If an error happens, we print it. If it is an `IntegrityError` and `allow_overwrite=False`, we also display the hint about setting `allow_overwrite=True`.